### DISCUSSION TITLE
User quota and policy template

### DISCUSSION THREAD

**Discussion by levindecaro** [minio/minio#14833] [Q&A]
**Is your feature request related to a problem? Please describe.**

In multi user environment, current minio implementation support external idm login, but it is lack of default policy assignment, there are few essential policy would be fair to included in future of minio.

1. default policy for new identities claims. ( default IAM policy )
2. user quota (such as, number of buckets, size, number of objects.)
3. assign policy template against different ldap groups.

**Describe the solution you'd like**
n/a

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

**Comment by harshavardhana** [comment]
There is no real explanation on what is needed here, it looks like a discussion moving this as a discussion.
